iotedgehubdev/composeproject.py

```diff
@@ -53,11 +54,19 @@ def compose(self):
                 self.Services[service_name]['labels'][self.edge_info['labels']] = ""
 
             try:
+                # Default restart policy is 'on-unhealthy'
+                # https://github.com/Azure/iotedge/blob/8bd573590cdc149c014cf994dba58fc63f1a5c74/edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Constants.cs#L18
+                restart_policy = config.get('restartPolicy', 'on-unhealthy')
```

Comment:
https://github.com/Azure/iotedge/blob/master/edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/IModule.cs#L66 there is also an "unknown" status. Better to handle it also.